index.html

```diff
@@ -17,5 +17,22 @@
       <a class="nav-item" href="#">ABOUT US</a>
       <a class="nav-item" href="#">CONTACT</a>
    </nav>
+   <div class="container-fluid">
```

Comment:
if I'm not mistaken, section also uses fixed container `.container` instead of `container-fluid`